pkg/utils/helpers.go

```diff
@@ -22,6 +23,22 @@ func ApplyMaskToStoredPbObject[T proto.Message](updateMask *fieldmaskpb.FieldMas
 	fieldmask.Update(updateMask, dst, src)
 }
 
+// Run function run the commands
+func Run(cmd []string, _ bool) (string, int) {
+	var out []byte
+	var err error
+	out, err = exec.Command(cmd[0], cmd[1:]...).CombinedOutput() //nolint:gosec
+	if err != nil {
+		/*if flag {
```

Comment:
if we don't need code, it can be just removed

Reply:
I will check on this

Reply:
Done